src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -8202,14 +8198,14 @@ public partial struct ManualResetValueTaskSourceCore<TResult>
         private TResult _result;
         private object _dummy;
         private int _dummyPrimitive;
-        public TResult GetResult(short token) { throw null; }
-        public ValueTaskSourceStatus GetStatus(short token) { throw null; }
-        public void OnCompleted(Action<object> continuation, object state, short token, ValueTaskSourceOnCompletedFlags flags) { throw null; }
-        public void Reset() { throw null; }
-        public bool RunContinuationsAsynchronously { get { throw null; } set { throw null; } }
-        public void SetException(System.Exception error) { throw null; }
-        public void SetResult(TResult result) { throw null; }
+        public bool RunContinuationsAsynchronously { get { throw null; } set { } }
```

Comment:
From https://github.com/dotnet/corefx/pull/33104

cc @stephentoub, please review for correctness